share/spice/leak_db/leak_db.js

```diff
@@ -23,8 +24,13 @@
                 sourceUrl: 'http://leakdb.abusix.com/?q='+ encodeURIComponent(query),
                 sourceName: 'Abusix'
             },
+            normalize: function(item) {
+                return {
+                    title: 'hashes for "' + api_result.query + '"'
```

Comment:
Perhaps a title like "LeakDB Hashes" and subtitle: "Key: MySuperNicePassword"?

/cc @abeyang -- any thoughts on the title design